client/http/client.go

```diff
@@ -217,14 +225,18 @@ func (ci *clientInner) doRequest(
 		logFields = append(logFields, zap.String("status", resp.Status))
 
 		bs, readErr := io.ReadAll(resp.Body)
+		var bodyStr string
 		if readErr != nil {
+			bodyStr = fmt.Sprintf("read body failed: %s", readErr.Error())
 			logFields = append(logFields, zap.NamedError("read-body-error", err))
 		} else {
+			// need to remove the last newline character
+			bodyStr = string(bytes.TrimRight(bs, "\n"))
```

Comment:
Why add this logic?

Reply:
For consistency with some ctl printed information